test/core/bitswap.spec.js

```diff
@@ -69,8 +67,8 @@ function connectNodes (remoteNode, inProcNode, callback) {
 let nodes = []
 
 function addNode (inProcNode, callback) {
-  df.spawn({
-    exec: './src/cli/bin.js',
+  dfProc.spawn({
+    exec: IPFS,
```

Comment:
@VictorBjelkholm this won't spawn the daemon, instead it will spawn a instance. //cc @dryajov

Reply:
@diasdavid I made the change.

Reply:
@richardschneider mind bringing it back to './src/cli/bin.js'? This is what is causing the issue that Victor is describing on -- https://github.com/ipfs/js-ipfs/pull/1103/files#r168933411 --. Instead of spawning a daemon with a WebSockets listener, this is now trying to spawn an IPFS instance with a WebSockets listener in the browser. Such thing is not possible.

Reply:
I see that @VictorBjelkholm has already done this.  What I was fixing was on Windows/Node.  Because `ipfsd-ctl` does a `hard kill`, sometimes file handles are left open for a while and the `repo cleanup` was failing.

PR [graceful stop](https://github.com/ipfs/js-ipfsd-ctl/pull/205) will fix this.  Do you mind looking at it.